Drs Gully System/README.md

```diff
@@ -0,0 +1,28 @@
+# Drs Gully System
+
+# About this Project
+One of the most treasured memories of growing up is playing cricket with friends. Just like me, there are countless other people whose favourite part of the day is to grab a pair of bat and ball and just play. Gully cricket is the most popular form of sport, is played in everywhere. So, today’s project is going to be very helpful for those cricket fans who want to create their own decision review system. 
+
+# Features
+The third empire takes the decision about no-ball, runout, or catch out. Here we are going to make the decision review system using python which will take an accurate decison aboutwheather the batsman is out or not out.
+
+# How to Run
+
+1. Run the installation script
```

Comment:
Umm, I think these instructions have nothing to do with this script. There is no file named `install.py` and no `Enter installation directory: project_dir`. Kindly check these and update according to the script pushed